service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/transport/amqps/AmqpFeedbackReceivedHandler.java

```diff
@@ -126,10 +89,9 @@ public void onDelivery(Event event)
             // Codes_SRS_SERVICE_SDK_JAVA_AMQPFEEDBACKRECEIVEDHANDLER_12_007: [The event handler shall settle the Delivery with the Accepted outcome]
             delivery.disposition(Accepted.getInstance());
             delivery.settle();
-          
-            // Codes_SRS_SERVICE_SDK_JAVA_AMQPFEEDBACKRECEIVEDHANDLER_12_008: [The event handler shall close the Session and Connection (Proton)]
-            recv.getSession().close();
-            recv.getSession().getConnection().close();
+
+            log.debug("Closing amqp feedback receiver link since a feedback message was received");
+            recv.close();
```

Comment:
Are we relying on link closure closing the session and associate connection? If yes, we should add appropriate comment here.

Reply:
Sure thing